packages/charts/src/Charts/SeverityLine/SeverityLine.js

```diff
@@ -0,0 +1,77 @@
+import './severity-line.scss';
+
+import React, { Component } from 'react';
+
+import PropTypes from 'prop-types';
+
+class SeverityLine extends Component {
+
+    constructor(props) {
+        super(props);
+        this._getSeverityValue = this._getSeverityValue.bind(this);
+        this.state = {
+            severity: 0
+        };
+    }
+
+    _getSeverityValue(value) {
+        const { config: { width } } = this.props;
+        const CHART_MAP = { 1: '1', 2: (width / 3), 3: (width * (2 / 3)), 4: (width - 1) };
+        this.setState({
+            severity: CHART_MAP[value]
+        });
+    }
+
+    componentDidMount() {
+        const { value } = this.props;
+        this._getSeverityValue(value);
+    }
+
+    render() {
+        const {
+            title,
+            value,
+            config: {
+                height,
+                width
+            }
+        } = this.props;
+        return <div className='ins-c-severity-line'>
+            <div className='ins-l-title'>
+                <span> {title} </span>
+            </div>
+            <div className='ins-l-chart'>
+                <svg height={height} width={width} viewBox={`0 0 ${width} ${height}`}>
+                    <polyline id='horizontal' points={`${1}, ${height / 2} ${width - 1}, ${height / 2}`}/>
```

Comment:
I'd prefer `className` over `id` as this prevents using this chart multiple times on same page.